dataplane/grpc/proto/cnibackend.pb.go

```diff
@@ -0,0 +1,376 @@
+// Code generated by protoc-gen-gogo. DO NOT EDIT.
```

Comment:
I think we may want to consider adding the tooling for contributors to regenerate this code, possibly adding this to the standard "make build" target. @caseydavenport, wdyt?